src/imitation/scripts/train_imitation.py

```diff
@@ -7,7 +7,7 @@
 from sacred.observers import FileStorageObserver
 from stable_baselines3.common import policies, utils, vec_env
 
-from imitation.algorithms.bc import BC
+from imitation.algorithms.bc import BC, reconstruct_policy
```

Comment:
Import modules not classes/methods: https://google.github.io/styleguide/pyguide.html#22-imports

This is an existing issue in this code, but one we should probably fix not exacebrate.

Reply:
Actually there is a problem with this; there is already a `bc` in the namespace, the function defined to call the command for behavioral cloning training from the script. (around line 175). I'll temporarily use `bc_algo` as the module alias to be able to import it, but feel free to suggest a better name.